event-based-simulators/main/event_based_simulators.py

```diff
@@ -368,12 +408,24 @@ def load(filename):
 # create managed object for every simulator
 [item.get_or_create_device_id() for item in simulators]
 
+
+#read & update Shiftplans
+SHIFTPLANS_MODELS = load("shiftplans.json")
+[oeeAPI.add_or_update_shiftplan(shiftplan) for shiftplan in SHIFTPLANS_MODELS]
+#first poll to fill the shiftplans array with shiftplans from locationsIds presented in the model
+[shiftplans.append(oeeAPI.get_shiftplan(shiftplan["locationId"], f'{datetime.utcnow():{shiftplan_dateformat}}', f'{datetime.utcnow()+shiftplan_polling_interval:{shiftplan_dateformat}}')) for shiftplan in SHIFTPLANS_MODELS]
```

Comment:
maybe use get_new_shiftplans() method to avoid code ducplication?

Reply:
Done